docs/src/tutorial/cython_tutorial.rst

```diff
@@ -331,6 +331,37 @@ everywhere. Adding types makes your code less readable, so use them with
 moderation.
 
 
+Primes with C++
+===============
+
+With Cython, it is also possible to take advantage of the C++ language, notably,
+part of the C++ standard library is directly importable from Cython code.
+
+Let's see what our :file:`primes.pyx` becomes when
+using `vector <http://en.cppreference.com/w/cpp/container/vector>`_ from the C++
+standard library.
+
+.. note::
+
+    Vector in C++ is a data structure which represents
+    a `stack <https://en.wikipedia.org/wiki/Stack_(abstract_data_type)>`_.
```

Comment:
I'd rather say "list or stack" here. Using a vector as a list seems like the most common use case to me. In fact, I'd write: "a data structure which implements a list or stack based on a resizeable C array. It is similar to the Python `array` type in the `array` standard library module."